docs/docs/platform/graphql/index.md

```diff
@@ -3,49 +3,58 @@ title: 'GraphQL'
 sidebar_position: 1
 ---
 
-Every Nhost app has its own autogenerated GraphQL API. The GraphQL API is based on the tables and columns in the [Postgres database](/platform/database) and is instantly available. It's [Hasura GraphQL engine](https://github.com/hasura/graphql-engine) that powers the GraphQL API.
+A GraphQL API is automatically and instantly available based on the tables and columns in your [database](/platform/database).
 
-The GraphQL API is available at: `https://[subdomain].nhost.run/v1/graphql`.
+The GraphQL API has instant support for inserting, selecting, updating, and deleting data, which usually account for 80% of all operations you need in your app.
+
+It's the [Hasura GraphQL engine](https://github.com/hasura/graphql-engine) that powers the GraphQL API which means that all documentation about [queries](https://hasura.io/docs/latest/graphql/core/databases/postgres/queries/index/), [mutations](https://hasura.io/docs/latest/graphql/core/databases/postgres/mutations/index/), and [subscriptions](https://hasura.io/docs/latest/graphql/core/databases/postgres/subscriptions/index/) from Hasura's documentation is applicable.
 
 ## What is GraphQL
 
-GraphQL is a query language for APIs that prioritize developer experience. The GraphQL API can be used to both fetch (query) and modify (mutation) data. GraphQL is especially powerful for frontend developers who wants to build products fast.
+GraphQL is a query language for APIs that prioritize developer experience. The GraphQL API can be used to both fetch (query) and modify (mutation) data. GraphQL is especially powerful for frontend developers who want to build products fast.
+
+GraphQL has grown rapidly in popularity in the last years and has been adopted by almost all major tech companies such as Facebook, GitHub, and Stripe.
+
+Building your GraphQL API is a lot of work, but with Nhost it's easy because every table and column is instantly available in your GraphQL API.
+
+## Endpoint
+
+The GraphQL API is available at `https://[subdomain].nhost.run/v1/graphql` When using the [CLI](/platform/cli) the GraphQL API is available at `http://localhost:1337/v1/graphql`.
+
+## GraphQL Clients for JavaScript
 
-### GraphQL clients for JavaScript
+The Nhost JavaScript client comes with a simple [GraphQL client](/reference/javascript/nhost-js/graphql) that works well for the backend or simple applications.
 
-To interact with the GraphQL API it's recommended to use a GraphQL client:
+When building a more complex frontend applications, we recommend using a more advanced GraphQL client such as:
 
 - [Apollo Client](https://www.apollographql.com/docs/react/)
```

Comment:
maybe it worths mentioning `@nhost/react-apollo` and `@nhost/apollo`

Reply:
Added two links to apollo for React and Vue

Reply:
I just added links to how to get started with Apollo Client and React/Vue for now.